### ISSUE TITLE
Abracadabra (Ethereum) Schema Version 1.2.1 Subgraph Version 1.0.1 Methodology Version 1.0.0 

### ISSUE THREAD

**Issue by pietydespises** [messari/subgraphs#721]
Things to fix: 
1. TVL requires tweeking on initial values and check spike in Oct-Nov
2. Staked Curve.fi USD-BTC-ETH Convex Deposit Abra Market TVL significant difference than what's quoted on site, inputTokenPriceUSD also NOT OK
3. yVault wETH TVL significant difference than what's quoted on site
4. FTT-MIM inputTokenPriceUSD - FTT significant difference from CoinGecko
5. wBTC and USDC all revenue metrics and inputTokenPrice metrics off.

Evidence: https://docs.google.com/spreadsheets/d/13VmW2X1NDmKbwRewaAKRTj5UloJiM36T/edit?usp=sharing&ouid=117022469384251277944&rtpof=true&sd=true

**Comment by lowjiansheng** [comment]
Too high TVL on Abracadabra Ethereum. 
Currently:
```{
    "data": {
        "lendingProtocols": [
            {
                "totalValueLockedUSD": "77932794955988730242069442.97085035"
            }
        ]
    }
}```

**Comment by dmelotik** [comment]
@lowjiansheng still iterating on this QA to try and fix... I will be working on this today.

**Comment by this-username-is-taken** [comment]
@pietydespises what's the latest on this? Dylan said he's finished all fixes

**Comment by ishraq8** [comment]
Fantom issue on wftm borrow vs tvl is resolved on currently syncing subgraph 1.2.9. Thank you @dmelotik, fyi @this-username-is-taken

**Comment by this-username-is-taken** [comment]
LFGGG!!
